src/core/linux/SDL_evdev_capabilities.h

```diff
@@ -33,12 +33,13 @@ typedef enum
 {
     SDL_UDEV_DEVICE_UNKNOWN = 0x0000,
     SDL_UDEV_DEVICE_MOUSE = 0x0001,
-    SDL_UDEV_DEVICE_KEYBOARD = 0x0002,
```

Comment:
We don't need to maintain internal ABI or symbol naming. Can you leave KEYBOARD alone and add HAS_KEYS after it, and change existing uses to HAS_KEYS (unless it actually means full keyboard?)

Reply:
Sure. The reason I added a new name is to make sure I caught all the locations where it was used - in fact they all want `HAS_KEYS`.